r-rse-index.Rmd

```diff
@@ -1,7 +1,6 @@
 ---
 title: "Merely Useful: RSE R"
 author: "Madeleine Bonsma-Fisher, Kate Hertweck, Damien Irving, Luke Johnston, Christina Koch, Sara Mahallati, Brandeis Marshall, Joel Ostblom, Elizabeth Wickes, Charlotte Wickham, and Greg Wilson"
-documentclass: Nemilov
```

Comment:
Looks good, except for the removal of this documentclass. It should be replaced with `book` after removing Nemilov. I'm pushing an update to the branch shortly.